db/migrate/20151123223902_add_permissions_to_list.rb

```diff
@@ -0,0 +1,5 @@
+class AddPermissionsToList < ActiveRecord::Migration
+  def change
+    add_column :permissions
```

Comment:
did this migration actually run? I don't think that it did. When you rake db:migrate, do you get an error? I think the problem lies here for the most part. Can you run the migration again and paste the output? Go find the docs for `add_column`, and see what else you need here.